### ISSUE TITLE
Numerical Representation of Velocity Arrows

### ISSUE THREAD

**Issue by nevilledusaj** [velocyto-team/velocyto.R#64]
Is there a location which the calculated velocity values of the projected arrows is stored in either the r or rvel.cd object that is created? If not, is there a way to have either the show.velocity.on.embedding.cor or gene.relative.velocity.estimates output an arrow direction and arrow length for each cell it calculates those values for?

**Comment by pkharchenko** [comment]
yes, use return.details=TRUE argument, and the arrow info will be returned in in the $garrows slot.

**Comment by nevilledusaj** [comment]
It doesn't seem to return anything for me when I add that argument in either the r environment or the rvel.cd list. Just looking through some of the source code I came across this line at the end of the section beginning with `if (return.details = TRUE)` section

`return(invisible(list(tp=tp,cc=cc,garrows=garrows,arrows=as.matrix(ars),vel=nd,eshifts=es,gvel=gv,geshifts=gs,scale=scale)))`

I am unfamiliar with how invisible objects work in R, but would this prevent it from being placed into the appropriate object?
